lib/web.js

```diff
@@ -87,7 +85,9 @@ var SolidWebClient = {
     return new Promise(function (resolve, reject) {
       var http = new XMLHttpRequest()
       http.open(method, url)
-      http.withCredentials = true
+      if (!options.noCredentials) {
+        http.withCredentials = true
```

Comment:
Do I read this correctly that by default the request does not set the credentials flag? I think it should be the other way around..

```
if (options.noCredentials) {
  http.withCredentials = false
}
```